library/navigation/src/main/java/ir/composenews/navigation/graph/FavoriteNews.kt

```diff
@@ -31,7 +31,7 @@ fun NavGraphBuilder.favoriteList(
             ContentType.DUAL_PANE -> {
                 val market =
                     entry.parcelableData<MarketModel>(Destinations.MarketDetailScreen().market)
-                        ?: (uiState.market as? MarketModel?)
+                        ?: uiState.market
```

Comment:
Why you remove this `as? MarketNodel?`?

Reply:
Because I changed `val market: Any? = null` to `val market: MarketModel? = null`, so this casting not require anymore.